static/lang/re-en.json

```diff
@@ -1980,7 +1980,11 @@
             "Monk": {
                 "OneInchPunch": {
                     "ThreeActionLabel": "One Inch Punch (Three Actions)",
-                    "TwoActionLabel": "One Inch Punch (Two Actions)"
+                    "TwoActionLabel": "One Inch Punch (Two Actions)",
+                    "Suboptions": {
+                        "TwoActions": "Two Actions",
+                        "ThreeActions": "Three Actions"
+                    }
```

Comment:
Let's move these up a few levels so they are just under SpecificRule to be more generic, since we can use them in other places.